domchanger.js

```diff
@@ -347,6 +346,9 @@ function processTag(array) {
   else {
     body = array.slice(1);
   }
+  if (Array.isArray(body[0]) && !body[0].length) {
+    body[0] = "";
```

Comment:
this is canceling it at the level of the parent node, so you wouldnt be removing the current one. it just prevents the children from being iterated. eg makes `["tbody", []]` treated like `["tbody", ""]`.